### PR TITLE
General fixes

### PR DESCRIPTION
**Pull request by nachosoloaga**

### CODE REVIEW

File: src/components/Menu.js [hunk 1]
```diff
@@ -17,6 +17,9 @@ const Menu = () => {
             </li>
           );
         })}
+        <li className="menu-list-item">📖 Clase 3</li>
+        <li className="menu-list-item">📖 Clase 4</li>
+        <li className="menu-list-item" style={{color:"gray"}}>📖 Clase 5</li>
```
Comment (JulianPasquale) [inline src/components/Menu.js:22]: I'd suggest to remove this... We did a proposal of a class 5 but it is not approved yet, so probably developers don't know that (and shouldn't)

File: src/slides/class1.jsx [hunk 2]
```diff
@@ -213,7 +213,7 @@ export default () => (
       </Text>
 
       <Text className="font-size-medium">
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:215]:
```suggestion
      <Text fontSize={30}>
```

File: src/slides/class1.jsx [hunk 3]
```diff
@@ -150,7 +150,7 @@ export default () => (
...
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:152]:
```suggestion
      <Text fontSize={30}>
```

File: src/slides/class1.jsx (hunk 2 above)
Comment (JulianPasquale) [reply src/slides/class1.jsx:215]: There are more cases like this, can we replace them all?

File: src/slides/class1.jsx [hunk 4]
```diff
@@ -278,38 +278,38 @@ export default () => (
     </Slide>
 
     {/* Primitives */}
-    <Slide>
+    <Slide backgroundColor="#0d0d0d">
       <Heading margin="0px" className="font-size-heading">
         Tipos de datos primitivos
       </Heading>
 
       <Text className="font-size-medium align-center" margin="0px" padding="0px">
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:286]: standarize this Text tag too

**Review by JulianPasquale — APPROVED** [review]

File: src/components/Menu.js (hunk 1 above)
Comment (nachosoloaga) [reply src/components/Menu.js:22]: Yes! I agree on that, I decided to write that down with a different font color but probably it's better to just delete it

**Pull request merged**